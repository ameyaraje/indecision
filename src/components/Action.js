import React from 'react';

const Action = (props) => {
    return (
        <div>
            <button className="big-button" onClick={props.handlePickOption} disabled={!props.hasOptions}>
                What do I do?
            </button>
        </div>
    );
};

export default Action;