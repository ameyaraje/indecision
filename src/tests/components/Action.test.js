import React from 'react';
import { shallow } from 'enzyme';

import Action from '../../components/Action';

test('should render Action component', () => {
    const wrapper = shallow(<Action />);

    expect(wrapper).toMatchSnapshot();
});