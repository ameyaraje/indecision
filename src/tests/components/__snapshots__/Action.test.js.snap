// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Action component 1`] = `
<div>
  <button
    className="big-button"
    disabled={true}
  >
    What do I do?
  </button>
</div>
`;
